i18n/utils.py

```diff
@@ -77,7 +77,9 @@ def _load_translations(cls, name, lang):
     def get_translated_field(cls, name, i18n_key, field, lang):
         translations = cls._load_translations(name, lang)
         try:
-            return translations[i18n_key][field]
+            # Always use keys to access the translations dict, since it's
+            # loaded from JSON
+            return translations[str(i18n_key)][str(field)]
```

Comment:
Does this make the `str(self.pk)` change above not needed?

Reply:
Yep! But I think that redundancy is valuable. This ensures that future models that override the i18n_key will continue to work even if they're not strings, and the `str(self.pk)` change ensures that the default case will use a string no matter what, which helps for consistency in the generated strings before and after they get passed through json